docs/server/push-duty.rst

```diff
@@ -40,6 +40,40 @@ This way we can ensure that:
 Making multiple releases of a package during a weekly milestone is totally
 fine since this helps with testing smaller sets of changes.
 
+Security Fixes
+++++++++++++++
+
+Security fixes for addons-server live on a separate private repository:
+
+    * `addons-server-security <https://github.com/mozilla/addons-server-security>`_
+
+When making a pull request against this repository, the remote branch should
+not be published to one's fork but to the repository itself, keeping the
+changes entirely private. Once the PR has been reviewed, it should *not* be
+merged right away. Instead, merging to master is part of push duty and happens
+right before tagging::
+
+  $ git checkout master
+  $ git pull
+  $ git fetch security
```

Comment:
I almost did but the doc about tagging didn't so I followed that and just added a note explaining what the remotes point to. Should I add something about adding the remote in the note ?